src/Altinn.AccessManagement.Core/Services/PolicyInformationPoint.cs

```diff
@@ -88,6 +88,10 @@ public async Task<List<Right>> GetRights(RightsQuery rightsQuery, bool returnAll
             {
                 policy = await _prp.GetPolicyAsync(org, app);
             }
+            else if (resourceMatchType == ResourceAttributeMatchType.Altinn2Service)
```

Comment:
These changes should be rolled back as GetRights is never to handle A2 rights lookup, which is handled by A2 and SBL Bridge.

Reply:
removed this part